### PR TITLE
[Sprint #6] MtoM module about Wait list & test code about MtoM module

### PR DESCRIPTION
**Pull request by jinwookim0**

### CODE REVIEW

**Comment by jinwookim0** [comment]
get specific member(s) from membership (mtom) should get
from Membership module. Because team instance could find
all members even if a member belonged not to that team.

File: buildbuild/buildbuild/settings.py [hunk 1]
```diff
@@ -122,3 +122,21 @@
...
+EMAIL_PORT = 465  #for submission 
+EMAIL_HOST_USER = "buildbuildteam@gmail.com"
+EMAIL_HOST_PASSWORD = os.environ['BUILDBUILD_PASSWORD']
+DEFAULT_FROM_EMAIL = "buildbuild@gmail.com"
+SERVER_EMAIL = "buildbuildteam@gmail.com"
+#DEFAULT_TO_EMAIL = 'to email'
+
+# The mailing contents for new user
+SUBJECT = " Welcome to buidlbuild team! "
+CONTENTS = "Hello~"
```
Comment (dobestan) [inline buildbuild/buildbuild/settings.py:140]: - this should be change later
- use html form for mail contents.

**Comment by dobestan** [comment]
> this implementation is really awesome

File: buildbuild/buildbuild/settings.py [hunk 2]
```diff
@@ -122,3 +122,21 @@
...
+
+# Defalut SMTP Host Setting
+
+EMAIL_BACKEND = 'django_smtp_ssl.SSLEmailBackend'
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 465  #for submission 
+EMAIL_HOST_USER = "buildbuildteam@gmail.com"
+EMAIL_HOST_PASSWORD = os.environ['BUILDBUILD_PASSWORD']
+DEFAULT_FROM_EMAIL = "buildbuild@gmail.com"
```
Comment (o8151814) [inline buildbuild/buildbuild/settings.py:134]: is it right?
i think 'buildbuildteam@gmail.com' <- it is right

**Pull request merged**